libc/config/linux/aarch64/entrypoints.txt

```diff
@@ -489,6 +489,7 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.rintl
     libc.src.math.round
     libc.src.math.roundf
+    libc.src.math.roundevenf
     libc.src.math.roundl
```

Comment:
I messed up the sorting. I enabled just this entrypoint because I use it in the performance test.